swift_scality_backend/server.py

```diff
@@ -33,6 +33,21 @@
 POLICY_STUB = object()
 
 
+class ScalityDiskFileRouter(object):
+    """
+    Replacement for Swift's DiskFileRouter object.
+    Always returns Scality's DiskFileManager implementation.
+
+    Copied from SwiftOnFile::obj/server.py
+    """
+    def __init__(self, *args, **kwargs):
+        self.manager_cls = swift_scality_backend.diskfile.DiskFileManager(
+            *args, **kwargs)
+
+    def __getitem__(self, policy):
```

Comment:
I am not sure I fully understand what you mean here.

And I would like to merge this PR soon because I think it is good as is and any refactoring is not 100% related to supporting expiring object.